LSDR/Assets/Scripts/Audio/MusicSystem.cs

```diff
@@ -42,10 +51,19 @@ private void setMetadataFromFilePath(string filePath)
         {
             var filename = Path.GetFileNameWithoutExtension(filePath);
             if (filename != null)
+            if (filename.Contains(" - "))
             {
-                var splitFilename = filename.Split(new[] {" - "}, 2, StringSplitOptions.RemoveEmptyEntries);
-                CurrentArtist = splitFilename[0];
-                CurrentSong = splitFilename[1];
+                {
+                   var splitFilename = filename.Split(new[] {" - "}, 2, StringSplitOptions.RemoveEmptyEntries);
+                   CurrentArtist = splitFilename[0];
+                   CurrentSong = splitFilename[1];
+                   return;
```

Comment:
This return statement is unnecessary, no extra code gets executed as the rest of the body of the function is covered by the 'else' statement